doc/changes/latest.inc

```diff
@@ -23,6 +23,7 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug with reading split files that have dashes in the filename by `Eduard Ort`_ (:gh:`8339`)
```

Comment:
I see you're a new contributor (hooray!) -- you need to add your name to `names.inc` with a link:

https://github.com/mne-tools/mne-python/blob/master/doc/changes/names.inc

We also now do some reformatting to bold the line for new contributors, see:

https://github.com/mne-tools/mne-python/blob/master/doc/changes/0.21.inc#L15